src/plugins/directives.js

```diff
@@ -1,19 +1,7 @@
 import Vue from 'vue';
 
 Vue.directive('focus', {
-  inserted(_el, _binding, vnode) {
-    const element = getElement(vnode);
-
-    if (element) {
-      element.focus();
-    }
+  inserted: function(el) {
```

Comment:
I like your decision to add to the `v-focus` directive but be careful when making modifications because it's applied to the root Vue instance. This change causes regressions when using `v-focus` with `labeled-input` in the add an image page.